metadata.yaml

```diff
@@ -20,4 +20,4 @@ resources:
     type: oci-image
     description: upstream docker image for MySQL
     auto-fetch: true
-    upstream-source: 'mysql:8.0.22'
+    upstream-source: "ubuntu/mysql:latest"
```

Comment:
It may be worth check with @jlounder and @jameinel what the recommended practice is - If we should be using `latest` or specific version.

Reply:
The available versions are these: https://hub.docker.com/r/ubuntu/mysql/tags?page=1&ordering=last_updated

Reply:
I meant fixing the version to a specific one rather than just picking the latest.